export const onError = (socket) => (error) => {
  try {
    //
  } catch (err) {
    //
  }
};
